### DISCUSSION TITLE
🔍 More Search 🔍

### DISCUSSION THREAD

**Discussion by LivingWithHippos** [LivingWithHippos/unchained-android#159] [Feedback]
### 🏷️ New features:

- [x] new search engine allows for hosting search too
- [x] added scroll to refresh on the user page (should fix token errors)

### 🔎  Enhancements:

- [x] a new search plugin
- [x] authentication should work better
- [x] added icons to download tabs

Notes:

- This version fix the crash on 4.13.4
- to avoid issues a new dev branch will be created, master will be used only for code of published releases
- new permissions were needed for fixing the crash reporter, they may be removed in the future

<hr /><em>This discussion was created from the release <a href='https://github.com/LivingWithHippos/unchained-android/releases/tag/4.18.4-beta'>🔍 More Search 🔍</a>.</em>

**Comment by anon-y** [comment]
Thanks for the rlsbb source! Can I ask if you can add scene-rls.net as a source down the road? I will also try to add it but my json skills are lacking

**Reply by LivingWithHippos** [reply]
Yes, it should be similar to the rlsbb one

**Comment by akogwapo14** [comment]
thank you sir

**Comment by boognish-rising** [comment]
I have tried with limited success.. er, no. Make that none success to help with adding some more plugins but truth be told I haven't invested the necessary time yet. I will put some time aside and despite not having much json experience under my belt, I think I should be able to use what you've provided thus far as a template.

Thanks for the killer app. Whether I'm successful or not, I'll be sure to throw you some dollars to support further development

**Reply by LivingWithHippos** [reply]
Thank you for your support. I have to admit that without at least a guide with images, plugin creation is going to be pretty hard. When I have time, maybe this weekend, I 'd like to fix the app and maybe also write that. 

JSON is just a data format so it's not hard at all, you need to know a little regex and html/javascript to be at ease
